.stylelintignore

```diff
@@ -0,0 +1,3 @@
+packages/core/scss
+packages/white-label/scss
+packages/cxl-lumo-styles/scss/typography.scss
```

Comment:
Smart :+1: yeah we can exclude upstream for time being, I don't see us doing much upstream commits in this branch.